ffi/Cargo.toml

```diff
@@ -4,6 +4,12 @@ version = "2.0.0-pre-rc.8"
 authors = ["Iroha 2 team <https://github.com/orgs/soramitsu/teams/iroha2>"]
 edition = "2021"
 
+[features]
+# Enables mutable references of non-robust transmutable types in FFI, non-transmutable are always disallowed.
+# When handing out non-robust mutable references across FFI, it's possible that the other
+# side will modify the given reference and store type's trap representation inside it.
+non_robust_ref_mut = []
```

Comment:
ok, out the window with it

Reply:
> Why do we need this as a feature?

actually, I made some changes and decided to leave this. It's has a minuscule footprint on the codebase.
I don't think we need it at the moment since we don't seem to share any mutable references to our smart contracts. Also some mutable references are always allowed and are not affected by this flag

> How likely are we to have people who don't want to use it?

I think it's more a question of how likely are we to have people that want to use it. All I can say is that I'm not sure